projects/plugins/jetpack/modules/notes.php

```diff
@@ -232,17 +232,19 @@ private static function get_notes_markup() {
 	 */
 	public function print_js() {
 		$link_accounts_url = is_user_logged_in() && ! ( new Connection_Manager( 'jetpack' ) )->is_user_connected() ? Jetpack::admin_url() : false;
-		?>
-<script data-ampdevmode type="text/javascript">
-/* <![CDATA[ */
-	var wpNotesIsJetpackClient = true;
-	var wpNotesIsJetpackClientV2 = true;
-		<?php if ( $link_accounts_url ) : ?>
-	var wpNotesLinkAccountsURL = '<?php echo esc_url( $link_accounts_url ); ?>';
-<?php endif; ?>
-/* ]]> */
-</script>
-		<?php
+		$script_contents   = <<<JS
```

Comment:
Let's switch to a nowdoc to ensure no accidental interpolation in here. WPCS will complain about it in the future.

```suggestion
		$script_contents   = <<<'JS'
```